server/embedded/src/main/resources/log4j2.xml

```diff
@@ -187,7 +187,7 @@ account managers so that we can coordinate edits with other customized copies of
         <!-- category for jasper messages -->
         <Logger name="org.apache.jasper" level="warn"/>
 
-        <Logger name="org.labkey" />
+        <Logger name="org.labkey" level="info" />
```

Comment:
the `<Root level="info">` above should make this (and the other two you added) moot.

Reply:
These were causing issues/errors when using the ConfigurationFactory, they don't seem to be doing so now. So I removed them.